src/main/java/com/dell/isg/smi/service/device/discovery/controller/DeviceDiscoveryController.java

```diff
@@ -36,9 +35,8 @@
 
     private static final Logger logger = LoggerFactory.getLogger(DeviceDiscoveryController.class.getName());
 
-
     @RequestMapping(value = "/range", method = RequestMethod.POST, headers = "Accept=application/json", consumes = "application/json", produces = "application/json")
-    @ApiOperation(value = "/range", nickname = "range", notes = "This operation allow user to collect server software identity throu wsman.", response = DiscoverdDeviceResponse.class, responseContainer = "List")
+    @ApiOperation(value = "/range", nickname = "range", notes = "This operation ping sweep and discovers the devices within the given range of IP.", response = DiscoverdDeviceResponse.class, responseContainer = "List")
```

Comment:
Change to :
This operation will ping sweep and discover devices within the given IP range.

Reply:
Done